File: app/src/main/java/com/rafamatias/nlp/presentation/model/Sentiment.java
package com.rafamatias.nlp.presentation.model;

public class Sentiment {
    public enum Type {
        HAPPY,
        NEUTRAL,
        SAD
    }

    private float score;
    private float magnitude;

    public Sentiment(float score, float magnitude) {
        this.score = score;
        this.magnitude = magnitude;
    }

    public Type getSentiment(){
        if(score < -0.5f){
            return Type.SAD;
        }else if(score > 0.2){
            return Type.HAPPY;
        }else{
            return Type.NEUTRAL;
        }
    }
}

File: domain/src/main/java/bloder/com/domain/models/sentiment/Sentiment.kt
package bloder.com.domain.models.sentiment

class Sentiment(private val score: Double) {

    fun getSentiment() : SENTIMENT = when {
        score < -0.2 -> SENTIMENT.SAD
        score < 0.5 -> SENTIMENT.NEUTRAL
        else -> SENTIMENT.HAPPY
    }
}

enum class SENTIMENT {
    HAPPY, NEUTRAL, SAD
}